language-plutus-core/bench/Bench.hs

```diff
@@ -111,7 +112,17 @@ main =
                       [ bench "valid" $ nf (fmap runCk) f'
                       , bench "invalid" $ nf (fmap runCk) g'
                       ]
+                , env evalFiles $ \ ~(f, g) ->
+                   let processor :: BSL.ByteString -> Either (Error AlexPosn) (Program TyName Name ())
+                       processor contents = void <$> (runQuoteT $ parseScoped contents)
+                       f' = processor f
+                       g' = processor g
+                   in
 
+                   bgroup "runCek"
+                     [ bench "valid" $ nf (fmap (unsafeRunCek mempty)) f'
+                     , bench "invalid" $ nf (fmap (unsafeRunCek mempty)) g'
+                     ]
```

Comment:
Merged benchmarks of `plutus-core-interpreter` into benchmarks of `language-plutus-core`.